OpenEdXMobile/src/main/java/org/edx/mobile/module/prefs/AppPrefs.kt

```diff
@@ -0,0 +1,65 @@
+package org.edx.mobile.module.prefs
+
+import android.content.Context
+import dagger.hilt.android.qualifiers.ApplicationContext
+import org.edx.mobile.view.BulkDownloadFragment
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Singleton
+class AppPrefs @Inject constructor(@ApplicationContext context: Context) :
```

Comment:
Please format it like this and leave a space before init method to increase code readability: 

```
class AppPrefs @Inject constructor(
    @ApplicationContext context: Context
) : PrefBaseManager(context, APP_INFO) {